### PR TITLE
workload: print time taken to import fixture

### PR DESCRIPTION
**Pull request by dt**
Release note: none.

### CODE REVIEW

**Review by danhhz — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/36059#-:-LapDv6I9LYuG2M_JRig:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @danhhz and @dt)

___
*[pkg/util/humanizeutil/humanize.go, line 119 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/36059#-LapDu9E5tyXF5CMxiQt:-LapDu9E5tyXF5CMxiQu:b-7m74ox) ([raw file](https://github.com/cockroachdb/cockroach/blob/60bf6faf66d7a366a1197fdc8b6d022deea68c6d/pkg/util/humanizeutil/humanize.go#L119)):*
> ```Go
> 
> // DataRate formats the passed byte count over duration as "x MiB/s".
> func DataRate(bytes int64, elapsed time.Duration) string {
> ```

nice!

File: pkg/util/humanizeutil/humanize.go [hunk 1]
```diff
@@ -113,3 +114,14 @@ func (b *BytesValue) String() string {
...
+
+// DataRate formats the passed byte count over duration as "x MiB/s".
+func DataRate(bytes int64, elapsed time.Duration) string {
+	if bytes == 0 {
+		return "0"
+	}
+	if elapsed == 0 {
+		return "inf"
+	}
+	return fmt.Sprintf("%0.2f MiB/s", (float64(bytes)/elapsed.Seconds())/float64(1<<20))
```
Comment (tbg) [inline pkg/util/humanizeutil/humanize.go]: One could imagine using `humanizeutil.IBytes` as `%s/s` to arrive at something like `1.4 GiB/s` once things got faster, but I'm happy hardcoding `MiB/s`

**Review by tbg — APPROVED** [review]
Wonderful.

File: pkg/util/humanizeutil/humanize.go (hunk 1 above)
Comment (danhhz) [reply pkg/util/humanizeutil/humanize.go]: Oh, yes, please do this now.

Comment (dt) [reply pkg/util/humanizeutil/humanize.go]: So I actually started with IBytes but I didn't actually like it switching between KiB and MiB -- it made it a little harder to compare across runs than always using a fixed unit.

Comment (danhhz) [reply pkg/util/humanizeutil/humanize.go]: Fair point. I'm happy to try it like this and switch to autoscaling units later if it ends up being the wrong tradeoff.

**Pull request merged**